module/notification/src/Resources/config/messenger.yaml

```diff
@@ -0,0 +1,4 @@
+framework:
+    messenger:
+        transports:
+            notification: '%ergonode.notification.messenger_transport%'
```

Comment:
In this time transport use only to send mail.

Reply:
yup. I'm not altering the behavior, just making it easier and more robust to use

Reply:
Ok. For these moments mailer uses this configuration but does not require this module.